src/translations/de.translation

```diff
@@ -5979,4 +6537,17 @@ Hinweis: Die Registerkarte &quot;NPSP-Einstellungen&quot; ist in der Nonprofit S
             <name>Contacts</name>
         </sections>
     </reportTypes>
-    </Translations>
+    <reportTypes>
```

Comment:
@ptewson-sfdo This section is one of the ones that we updated the cci task to strip out in the main translation branch. I suspect you can either manually edit out this content from the `.translation` files are copy the updated cci task into this branch and re-run the task (just not committing the updated yml file).